lib/Mail/Milter/Authentication/Handler/DKIM.pm

```diff
@@ -70,6 +70,17 @@ sub header_callback {
 
     if ( lc($header) eq 'dkim-signature' ) {
         $self->{'has_dkim'} = 1;
+        my ($domain) = $value =~ /d=([^;]*);/;
```

Comment:
The regex for pulling these out is less than ideal, and could be confused by wrapping.